drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -433,7 +433,7 @@ static int _set(netdev2_t *netdev, netopt_t opt, void *val, size_t len)
                 uint8_t chan = ((uint8_t *)val)[0];
                 if (chan < AT86RF2XX_MIN_CHANNEL ||
                     chan > AT86RF2XX_MAX_CHANNEL) {
-                    res = -ENOTSUP;
+                    res = -EINVAL;
                     break;
                 }
                 at86rf2xx_set_chan(dev, chan);
```

Comment:
I know GitHub is blending it out, but it is commented on in the line below. This PR does not change this behavior since the `chan` field in the netdev2_ieee802154_t struct still needs to be set.